web/client/components/widgets/widget/enhancedWidgets.js

```diff
@@ -0,0 +1,121 @@
+import { compose } from 'recompose';
+
+// enhancers for base menus and functionalities
+import chartWidget from '../enhancers/chartWidget';
+import counterWidget from '../enhancers/counterWidget';
+import tableWidget from '../enhancers/tableWidget';
+import legendWidget from '../enhancers/legendWidget';
+import textWidget from '../enhancers/textWidget';
+import mapWidget from '../enhancers/mapWidget';
+
+// Enhancers for ajax support
+import wpsChart from '../enhancers/wpsChart';
+import wpsCounter from '../enhancers/wpsCounter';
+import wfsTable from '../enhancers/wfsTable';
+
+
+// enhancers for dependencies management
+import dependenciesToFilter from '../enhancers/dependenciesToFilter';
+import dependenciesToOptions from '../enhancers/dependenciesToOptions';
+import dependenciesToWidget from '../enhancers/dependenciesToWidget';
+import dependenciesToExtent from '../enhancers/dependenciesToExtent';
+import dependenciesToLayers from '../enhancers/dependenciesToLayers';
+import dependenciesToMapProp from '../enhancers/dependenciesToMapProp';
+
+
+import BaseChartWidget from './ChartWidget';
+import BaseTextWidget from './TextWidget';
+import BaseMapWidget from './MapWidget';
+import BaseTableWidget from './TableWidget';
+import BaseCounterWidget from "./CounterWidget";
```

Comment:
align quotes, sometimes they have " some other '

maybe we can think of an eslint rule it it exists